2800_app.js

```diff
@@ -304,6 +318,7 @@ app.post("/adminEditUser", function (req, res) {
         WHERE ID = ` + req.body.userID;
 
         con.query(editUser, function (error, results) {
+            con.end(err => {if (err) {console.log(err)}});

```

Comment:
why was this added? this will hard crash our app if any admin tries to edit any user.

Reply:
Removed this code.